internal/resources/examples_test.go

```diff
@@ -82,6 +82,10 @@ func TestAccExamples(t *testing.T) {
 				testutils.CheckCloudInstanceTestsEnabled(t)
 			},
 		},
+		{
+			category:  "Connections",
+			testCheck: func(t *testing.T, filename string) {},
```

Comment:
I think I need to include this new resource category here, but I'm not sure what to put in `testCheck`...